server/aws-lsp-codewhisperer/src/language-server/chat/chatController.ts

```diff
@@ -2,7 +2,7 @@ import {
     GenerateAssistantResponseCommandInput,
     GenerateAssistantResponseCommandOutput,
 } from '@amzn/codewhisperer-streaming'
-import { TabChangeParams, chatRequestType } from '@aws/language-server-runtimes/protocol'
+import { LSPErrorCodes, TabChangeParams, chatRequestType } from '@aws/language-server-runtimes/protocol'
```

Comment:
Why do we use LSPErrorCodes, for simplicity? If they don't provide a value I would argue we shouldn't use them

Reply:
This makes handling exception a lot easier at least on VS side because the LSP library convert these into meaningful exceptions